game-app/game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindowModel.java

```diff
@@ -29,6 +32,24 @@ Optional<Path> getInstallLocation(final MapDownloadItem mapDownloadItem) {
     return installedMapsListing.findMapFolderByName(mapDownloadItem.getMapName());
   }
 
+  public Optional<@NonNls String> getByteSizeTextForMap(final MapDownloadItem mapDownloadItem) {
```

Comment:
Should return a number here. Returning a string is mixing the UI and model logic together.

Reply:
What with exception case?
Maybe you provide a commit for this to show me what you think is best.

Reply:
Change: 

```
byteSizeText =
                    org.apache.commons.io.FileUtils.byteCountToDisplaySize(
                        org.triplea.io.FileUtils.getByteSizeFromPath(mapPath));
```

To:

```
   return                    org.apache.commons.io.FileUtils.byteCountToDisplaySize(
                        org.triplea.io.FileUtils.getByteSizeFromPath(mapPath));

```

Reply:
I think this is resolved now, right?